github_commands.txt

```diff
@@ -3,7 +3,7 @@ git add - adds files and folders to stage
 git commit -m "" - commits the changes with a message
 git status - shows the status of changes of files
 git add remote origin - adds the origin of the repo to the project
-git push -u origin main - pushs the changes to repository
+git push -u origin main - pushes the changes to repository
```

Comment:
Nurlan, your change is great, but, don't forget to make the first letter in uppercase